src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,18 @@
+@use 'sass:string';
+@use 'sass:math';
+
+$button-color: #00acdc;
+$main-text-color: #060b35;
+$second-text-color: #616070;
+$price-in-grn: 69999;
+
+@function converter($sum) {
+  $factor: 1000;
+
+  @return math.div(round(($sum / 31832.19) * $factor), $factor);
+}
+
+$price-in-dollars: converter($price-in-grn);
+$result-first-part: string.slice($price-in-dollars + ' ', 0, 1) + ',';
+$result-second-part: string.slice($price-in-dollars + ' ', 3);
+$price-in-string: $result-first-part + $result-second-part;
```

Comment:
The logic for creating `$price-in-string` assumes a specific format for `$price-in-dollars`. Ensure that `$price-in-dollars` is always a string with a length that matches these operations, or this could lead to unexpected results.